modules/imgproc/src/undistort.avx2.cpp

```diff
@@ -69,7 +69,7 @@ int initUndistortRectifyMapLine_AVX(float* m1f, float* m2f, short* m1, ushort* m
     const __m256d __matTilt_12 = _mm256_set1_pd(matTilt[5]);
     const __m256d __matTilt_22 = _mm256_set1_pd(matTilt[8]);
 
-    for (; j <= width - 4; j += 4, _x += 4 * ir[0], _y += 4 * ir[3], _w += 4 * ir[6])
+    for (; j < width - 4; j += 4, _x += 4 * ir[0], _y += 4 * ir[3], _w += 4 * ir[6])
```

Comment:
> Should be less than width - 4. If width = 4, the for loop should be executed once only.

Actually if width == 4 then this loop will not be executed at all (`0 < 4 - 4` is false).

Reply:
Sorry I was dreaming. I will change in a bit as I don't have access to a dev machine.